.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
           deno-version: v2.x
       - name: Install Deno requirements
         run: |
-          deno install
+          python pnpm.py install --frozen-lockfile
       - name: Build control bundle
         run: |
           deno task bundle
```

Comment:
note that this one should stay as `deno task bundle` since it is the control; the `pnpm.py` generated bundles should be tested against it